main.tf

```diff
@@ -592,3 +595,54 @@ resource "aws_config_config_rule" "iam_mfa_for_console_access" {
     "aws_lambda_permission.iam_mfa_for_console_access",
   ]
 }
+
+### RESTRICTED COMMON PORTS: ACCESS
+resource "aws_config_config_rule" "restricted_common_ports_access" {
+  count = "${var.create_config_rules && ! local.exclude_restricted_common_ports_access ? 1 : 0}"
+
+  name             = "restricted-common-ports-access"
+  description      = "Checks whether security groups that are in use disallow unrestricted incoming TCP traffic to the specified ports."
+  input_parameters = "{\"blockedPort1\":\"22\",\"blockedPort2\":\"3389\"}"
+
+  scope {
+    compliance_resource_types = ["AWS::EC2::SecurityGroup"]
+  }
+
+  source {
+    owner             = "AWS"
+    source_identifier = "RESTRICTED_INCOMING_TRAFFIC"
+  }
+}
+
+### RESTRICTED COMMON PORTS: DATABASE
+resource "aws_config_config_rule" "restricted_common_ports_database" {
+  count = "${var.create_config_rules && ! local.exclude_restricted_common_ports_database ? 1 : 0}"
+
+  name             = "restricted-common-ports-database"
+  description      = "Checks whether security groups that are in use disallow unrestricted incoming TCP traffic to the specified ports."
+  input_parameters = "{\"blockedPort1\":\"1433\",\"blockedPort2\":\"1521\",\"blockedPort2\":\"3306\",\"blockedPort2\":\"4333\",\"blockedPort2\":\"5432\"}"
+
+  scope {
+    compliance_resource_types = ["AWS::EC2::SecurityGroup"]
+  }
+
+  source {
+    owner             = "AWS"
+    source_identifier = "RESTRICTED_INCOMING_TRAFFIC"
+  }
+}
+
+### EBS SNAPSHOT PUBLIC RESTORABLE
+resource "aws_config_config_rule" "ebs_snapshot_public_restorable_check" {
+  count = "${var.create_config_rules && ! local.exclude_ebs_snapshot_public_restorable_check ? 1 : 0}"
+
+  name                        = "ebs-snapshot-public-restorable-check"
+  description                 = "Checks whether Amazon Elastic Block Store (Amazon EBS) snapshots are not publicly restorable. The rule is NON_COMPLIANT if one or more snapshots with RestorableByUserIds field are set to all, that is, Amazon EBS snapshots are public."
```

Comment:
Remove the `not` from from snapshots are not publicly restorable

Reply:
if "not publicly restorable" then "compliant"; that's the language direct from the AWS managed rule and I think it makes sense, even if a bit odd to follow.

Reply:
The wording is weird to me and doesn't read cleanly.

Reply:
It's straight from the doc, I think we should just take it rather than worry about the language and grammar too much. If this amazon doc is on github, we can open a pr there...

https://docs.aws.amazon.com/config/latest/developerguide/ebs-snapshot-public-restorable-check.html